padd.sh

```diff
@@ -267,7 +269,9 @@ GetSystemInformation() {
   fi
 
   # Device model
-  if [ -f /sys/firmware/devicetree/base/model ]; then
+  if [ -f /sys/devices/virtual/dmi/id/product_name ]; then
```

Comment:
On my system `product_family` would be the better choice.

```
chrko@ThinkPad-X230:~$ cat /sys/devices/virtual/dmi/id/product_name 
2325CN3
cat /sys/devices/virtual/dmi/id/product_family 
ThinkPad X230


```

Reply:
I used the same file used by applications like neofetch to show the `host name`. I tried with an MSI and a Mac Mini and `product_name` and the results are these:
```
cat /sys/devices/virtual/dmi/id/product_family 
PS
cat /sys/devices/virtual/dmi/id/product_name 
PS63 Modern 8SC

cat /sys/devices/virtual/dmi/id/product_family
Macmini
cat /sys/devices/virtual/dmi/id/product_name
Macmini6,2
```
While on an Asus laptop the situation is as your:
```
cat /sys/devices/virtual/dmi/id/product_family
VivoBook
cat /sys/devices/virtual/dmi/id/product_name
X510UQR
```

So, I thought that a possible solution could be if `product_family` is a substring of `product_name` then only the latter is shown, otherwise both the strings are displayed.

What do you think?

Reply:
Let me know if it works also in your system

Reply:
It works. 

```
chrko@ThinkPad-X230:~$ ./test.sh 
ThinkPad X230 2325CN3
```